baybe/utils/optional.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for handling optional dependencies."""
+
+import importlib
+import importlib.util
+import warnings
+from typing import Any, Literal, Optional
+
+
+def import_optional_module(
```

Comment:
is this importing the entire module or `something` from the module? If the latter then a rename to `import_from_optional_module` or `optional_import` or similar might be better

the doc string says import the module, but then theres the `attribute` arg so to me ti seems theres more to what this function does